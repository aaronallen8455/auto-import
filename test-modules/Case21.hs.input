module Case21 where

type X = () :<|> ()

main :: IO ()
main = pure ()
