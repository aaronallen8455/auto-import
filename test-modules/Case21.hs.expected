module Case21 where

import Servant.API ((:<|>)((:<|>)))

type X = () :<|> ()

main :: IO ()
main = pure ()
